pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -194,7 +250,9 @@ class CollectionViewSet(
     filterset_class = CollectionFilter
     pagination_class = LimitOffsetPagination
 
-    DEFAULT_ACCESS_POLICY = _PERMISSIVE_ACCESS_POLICY
+    queryset_filtering_required_permission = "ansible.view_ansiblerepository"
```

Comment:
We should only require users to have view permissions on the repository when the repository is private.